cmake/Platform/Mac/PreInstallSteps_mac.cmake.in

```diff
@@ -28,10 +26,11 @@ elseif (EXISTS @CMAKE_BINARY_DIR@/bin/debug/O3DE_SDK)
 elseif (EXISTS @CMAKE_BINARY_DIR@/bin/release/O3DE_SDK)
     set(sdk_launcher_config release)
 endif()
-file(COPY @CMAKE_BINARY_DIR@/bin/${sdk_launcher_config}/O3DE_SDK
+file(INSTALL @CMAKE_BINARY_DIR@/bin/${sdk_launcher_config}/O3DE_SDK
     DESTINATION ${CMAKE_INSTALL_PREFIX}/O3DE_SDK.app/Contents/MacOS
+    USE_SOURCE_PERMISSIONS
 )
-file(COPY @CMAKE_BINARY_DIR@/runtime_install/${sdk_launcher_config}/ProjectManagerBundlePath.setreg
+file(INSTALL @CMAKE_BINARY_DIR@/runtime_install/${sdk_launcher_config}/O3DE_SDK_InstallPath.setreg
```

Comment:
To make sure this file is only loaded in the `O3DE_SDK` launcher can you add a specialization tag of `o3de_sdk` to the filename.
So that it becomes
```suggestion
file(INSTALL @CMAKE_BINARY_DIR@/runtime_install/${sdk_launcher_config}/O3DE_SDK_InstallPath.o3de_sdk.setreg
```

With that, we can then updated the O3DE_SDK executable to add that specialization when loading the .setreg files, so only that applciations loads it.

Reply:
Fixed.